README.md

```diff
@@ -66,41 +65,36 @@ To start a new Autoabr job send a `POST` request to the `/autoabr` endpoint with
 
 ```json
 {
-   "encodingSettingsUrl": "s3://vmaf-files/encoding-profile-h265.json",
-   "pipelineUrl": "s3://vmaf-files/pipeline.json",
-   "job": {
-      "name": "job-name",
-      "reference": "s3://vmaf-files/tv2-vod-references/reference.mov",
-      "models": [
-        "UHD"
-      ],
-      "bitrates": [
-        10000000,
-        12800000
-      ],
-      "resolutions": [
-        {
-          "width": 768,
-          "height": 432
-        },
-        {
-          "width": 1280,
-          "height": 720
-        },
-        {
-          "width": 1920,
-          "height": 1080
-        },
-        {
-          "width": 2560,
-          "height": 1440
-        },
-        {
-          "width": 3840,
-          "height": 2160
-        }
-      ]
-   }
+  "encodingSettingsUrl": "s3://vmaf-files/encoding-profile-h265.json",
+  "pipelineUrl": "s3://vmaf-files/pipeline.json",
+  "job": {
+    "name": "job-name",
+    "reference": "s3://vmaf-files/tv2-vod-references/reference.mov",
```

Comment:
Should we remove references to clients (tv2) here?

Reply:
Good point. Yes, we should. I'll fix it.